### PR TITLE
Added loglevel argument

### PR DESCRIPTION
**Pull request by maaktweluit**
Enabled `--loglevel` as argument and replace the following objects with the new level:
- ROOT logger
- handlers
- loggers

Possible values are `WARNING`, `INFO` and `DEBUG`

Also fixed some linting errors along the way

### CODE REVIEW

**Review by Krigpl — APPROVED** [review]

File: golem/core/common.py [hunk 1]
```diff
@@ -171,18 +171,30 @@ def config_logging(suffix='', datadir=None):
...
+        if loglevel:
+            handler['level'] = loglevel
         if 'filename' in handler:
             handler['filename'] %= {
                 'logdir': str(logdir_path),
                 'suffix': suffix,
             }
 
+    if loglevel:
+        for _logger in list(LOGGING.get('loggers', {}).values()):
```
Comment (mfranciszkiewicz) [inline golem/core/common.py]: Conversion to `list` is superfluous

File: golem/core/common.py [hunk 2]
```diff
@@ -4,12 +4,13 @@
...
 from multiprocessing import cpu_count
 import os
-from pathlib import Path
-import pytz
 import sys
+import pytz
 
 from golem.core import simpleenv
 
+logger = logging.getLogger(__name__)
```
Comment (mfranciszkiewicz) [inline golem/core/common.py]: Could we avoid using loggers prior to configuring them?

**Review by mfranciszkiewicz — CHANGES_REQUESTED** [review]

File: golem/core/common.py (hunk 2 above)
Comment (maaktweluit) [reply golem/core/common.py]: Moved log message to after the config is applied, good catch!

File: golem/core/common.py (hunk 1 above)
Comment (maaktweluit) [reply golem/core/common.py]: Indeed, removed it and everything seems to still work :)

**Review by mfranciszkiewicz — APPROVED** [review]

**Pull request merged**